keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/artifact/ImageHandler.kt

```diff
@@ -146,7 +145,7 @@ class ImageHandler(
         user = serviceAccount,
         application = application,
         notifications = emptySet(),
-        subject = subject,
+        subject = "bakery:image:$artifact.name",
```

Comment:
Everywhere else this is a resource id so I thought I'd make it clearly something consistent with that